### PR TITLE
RNAblueprint

### PR DESCRIPTION
**Pull request by j-4**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/rnasketch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  #git_rev: 1.4
+  #git_url: https://github.com/ViennaRNA/RNAsketch.git
+
+build:
+  skip: True  # [not py27 or osx]
+  number: 0
+
+requirements:
+  build:
+    -  python 2.*
```
Comment (bgruening) [inline recipes/rnasketch/meta.yaml]: please don't pin python this way, you skip it already above

File: recipes/rnasketch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package:
+  name: rnasketch
+  version: "1.4.0"
+
+source:
+  fn: RNAsketch-1.4.tar.gz
+  url: https://github.com/ViennaRNA/RNAsketch/archive/1.4.tar.gz
+  md5: 3886fdc11dbd53349de260ec45a42358
+  #git_rev: 1.4
```
Comment (bgruening) [inline recipes/rnasketch/meta.yaml]: can you please remove this comments

File: recipes/rnablueprint/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+build:
+  skip: True  # [not py27 or osx]
+  number: 0
+
+requirements:
+  build:
+    #-  libtool
+    -  boost >=1.55.0
+    -  swig >=3.0.7
+    -  python 2.*
```
Comment (bgruening) [inline recipes/rnablueprint/meta.yaml]: do not pin python like that, use build/skip

File: recipes/rnablueprint/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package:
+  name: rnablueprint
+  version: "1.2.0"
+
+source:
+  fn: RNAblueprint-1.2.tar.gz
+  url: https://github.com/ViennaRNA/RNAblueprint/releases/download/1.2/RNAblueprint-1.2.tar.gz
+  md5: 3d02264f8d0dd1d15d003f4b25aafeeb
+  #git_rev: 1.2
```
Comment (bgruening) [inline recipes/rnablueprint/meta.yaml]: please remove

File: recipes/rnablueprint/build.sh [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+#sh autogen.sh
+
+./configure \
+  --prefix=$PREFIX \
+  --disable-perl \
+  --disable-doxygen-doc \
+  --disable-program \
+  --disable-tests 
+
+make -j
```
Comment (bgruening) [inline recipes/rnablueprint/build.sh]: can you run `make test`?

File: recipes/rnablueprint/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+requirements:
+  build:
+    #-  libtool
+    -  boost >=1.55.0
+    -  swig >=3.0.7
+    -  python 2.*
+
+  run:
+    -  boost >=1.55.0
```
Comment (eggzilla) [inline recipes/rnablueprint/meta.yaml]: you can add boost like this:
 - boost {{CONDA_BOOST}}*

Comment (eggzilla) [reply recipes/rnablueprint/meta.yaml]: maybe you need change the LD_LIBRARY_PATH like this:
export LD_LIBRARY_PATH="$PREFIX/lib"

File: recipes/rnablueprint/build.sh (hunk 5 above)
Comment (j-4) [reply recipes/rnablueprint/build.sh]: I included `make check -j` aafter `make -j` now

**Comment by j-4** [comment]
Thanks for the review, I included all your comments!

**Comment by eggzilla** [comment]
mhm, macos is still not triggering, we have to wait until one of the owners adds you.
Please trigger a rebuild then

**Comment by bgruening** [comment]
Cool, thanks @j-4!

**Pull request merged**